tools/operator/launcher/networkspec.yaml

```diff
@@ -37,60 +37,60 @@ orderer:
   batchtimeout: 2s
```

Comment:
Is there a reason why you chose NOT to edit these to use camelCase too?

Reply:
No particular reason. I think I forgot about that one. Wil change

Reply:
No particular reason. I think I forgot about it. Will change